src/main/java/com/manardenza/servise/HotelService.java

```diff
@@ -25,11 +26,11 @@ public static HotelService getInstance() {
     }
 
     public List<Hotel> findHotelByName(String hotelName) {
-        return null;
+        return new ArrayList<Hotel>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2293)